scripts/measures/2023/update-measures-util.ts

```diff
@@ -132,8 +128,18 @@ export function updateMeasuresWithChangeFile(
             exports.deleteMeasure(acceptedDeleteStack[i].measureId, acceptedDeleteStack[i].category, measuresJson);
         }
 
-        exports.updateChangeLog(fileName, changesPath);
-        info(`File '${fileName}' successfully ingested into measures-data ${performanceYear}`);
+        //ingest changed measures
```

Comment:
Moved changes to after additions and deletions so we can have the very latest version of the measures data before validating changes.